clients/venice-thin-client/src/main/java/com/linkedin/venice/client/stats/ClientStats.java

```diff
@@ -110,87 +106,141 @@ protected ClientStats(MetricsRepository metricsRepository, String storeName, Req
     Rate retryRequestSuccessKeyCount = new Rate();
     retryRequestSuccessKeyCountSensor =
         registerSensor("retry_request_success_key_count", retryRequestSuccessKeyCount, new Avg(), new Max());
-    retryKeySuccessRatioSensor = registerSensor(
```

Comment:
Is it removed by intention for all stores?

Reply:
same